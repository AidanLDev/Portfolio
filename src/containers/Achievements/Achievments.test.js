import React from 'react';
import Achievemnts from './Achievements';
import { shallow } from 'enzyme';

describe('<Achievements />', () => {
  it('Renders Achievemnts Component Correctly', () => {
    const component = shallow(<Achievemnts />);
    expect(component).toMatchSnapshot();
  });
});
