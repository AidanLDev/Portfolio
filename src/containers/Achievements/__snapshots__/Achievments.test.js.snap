// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Achievemnts Component Correctly 1`] = `
<div>
  <Skills />
  <hr />
  <h1>
    Certifications
  </h1>
  <div
    className="achCards"
  >
    <div
      className="degreeCard"
    >
      <Card
        img="degree.jpeg"
        imgHeight={400}
        imgWidth={300}
        title="BSc Computing"
      />
    </div>
    <Card
      content={
        <div
          className="AWS"
        >
          <p>
            May 2019, I achieved my AWS cloud practitioner certification. Demonstrating an overall understanding of the AWS Cloud along with the following skills:
          </p>
          <ul>
            <li>
              AWS cloud architectural principles
            </li>
            <li>
              AWS cloud value proposition
            </li>
            <li>
              Key AWS services
            </li>
            <li>
              Security and compliance aspects of the AWS platform and the shared security model
            </li>
            <li>
              Billing, account management, and pricing models
            </li>
          </ul>
          <p>
            Continuing on the AWS certification learning path, working on the AWS Solutions Architecture
          </p>
          <div
            className="tooltip"
          >
            <span
              className="tooltiptext"
            >
              View Badge
            </span>
            <a
              href="https://www.certmetrics.com/amazon/public/badge.aspx?i=9&t=c&d=2019-03-14&ci=AWS00694170"
              rel="noopener noreferrer"
              target="_blank"
            >
              <ForwardRef(MotionComponent)
                alt="AWS Cloud Practitioner badge"
                src="CloudPractitionerEmailBadge.png"
                whileTap={
                  Object {
                    "scale": 0.8,
                  }
                }
              />
            </a>
          </div>
        </div>
      }
      title="AWS"
    />
    <Card
      content={
        <div
          className="react16"
        >
          <img
            alt="React-16 Cert"
            height="200"
            src="udemyBurgerBuilder.jpg"
            width="400"
          />
          <p>
            Maximilian Schwarzm√ºller's comprehensive React 16 course. Gained indepth knowledge covering the following moduals:
          </p>
          <ul>
            <li>
              Ajax | Accessing a server | Routing | Forms/Validation | Redux | Testing | Next.js | Animations | Redux Saga | Hooks
            </li>
          </ul>
        </div>
      }
      title="Udemy"
    />
    <Card
      content={
        <div
          className="padding"
        >
          <p>
            Completed the 'Stack JavaScript Developer' track, gaining relevant skills in becoming a competent JavaScript developer.
          </p>
        </div>
      }
      img="teamTreeHouse.jpg"
      imgHeight={200}
      imgWidth={400}
      title="Team Treehouse"
    />
  </div>
  <hr />
  <h1>
    Projects
  </h1>
</div>
`;
